src/java/org/apache/cassandra/service/StorageProxy.java

```diff
@@ -1860,7 +1866,11 @@ private static float calculateResultRowsUsingEstimatedKeys(ColumnFamilyStore cfs
                     // collect replies and resolve according to consistency level
                     RangeSliceResponseResolver resolver = new RangeSliceResponseResolver(nodeCmd.keyspace, command.timestamp);
                     List<InetAddress> minimalEndpoints = filteredEndpoints.subList(0, Math.min(filteredEndpoints.size(), consistency_level.blockFor(keyspace)));
-                    ReadCallback<RangeSliceReply, Iterable<Row>> handler = new ReadCallback<>(resolver, consistency_level, nodeCmd, minimalEndpoints);
+                    ReadCallback<RangeSliceReply, Iterable<Row>> handler = new ReadCallback<>(resolver,
```

Comment:
nit (can ignore): any reason why we don't use the other constructor

Reply:
The other one uses a couple different parameters (blockFor and keyspace) so I figured passing Optional.empty() here was cleaner than overloading the constructor again